tools/postReleaseScripts/publishGHRelease.ps1

```diff
@@ -3,10 +3,17 @@
 
 #requires -Version 6.0
 
+[CmdletBinding()]
```

Comment:
Is this one of the scripts we're supposed to run? If so, can you provide a comment with an invocation of it?

Reply:
Will do